packages/server/.env.dev

```diff
@@ -33,12 +31,12 @@ APP_LOG_LEVEL: "verbose"
 # Email (only one of the api provider should be defined)
 #
 
-EMAIL_SENDER: "test@example.com" # This is only needed when SENDGRID_API_KEY is defined
+# EMAIL_SENDER: "test@example.com" # This is only needed when SENDGRID_API_KEY is defined
 
-SENDGRID_API_KEY: "SG.1234567890qwerty"
+# SENDGRID_API_KEY: "SG.1234567890qwerty"
```

Comment:
Are there other options than sendgrid, like local postfix or custom smpt credentials?

Reply:
We decided to only focus on sendgrid for the MVP (because of the big free tier and it's probably what JSG will end up using) but there is already an issue to add more options soon after: #4285 (which I just updated).